tests/cypress/integration/actions_objects2/case_108_rotated_bounding_boxes.js

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { labelName, taskName } from '../../support/const';
+
+context('Rotated bounding boxes.', () => {
+    const caseId = '108';
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+    const createRectangleTrack2Points = {
+        points: 'By 2 Points',
+        type: 'Track',
+        labelName,
+        firstX: createRectangleShape2Points.firstX,
+        firstY: createRectangleShape2Points.firstY - 150,
+        secondX: createRectangleShape2Points.secondX,
+        secondY: createRectangleShape2Points.secondY - 150,
+    };
+
+    const transformMatrixShape = [];
+
+    function testShapeRotate(shape, x, y, expectedRorate, shift) {
+        cy.get(shape)
+            .trigger('mousemove')
+            .trigger('mouseover')
+            .should('have.class', 'cvat_canvas_shape_activated');
+        cy.get('.svg_select_points_rot')
+            .should('be.visible')
+            .and('have.length', 1)
+            .trigger('mousemove')
+            .trigger('mouseover')
+            .trigger('mousedown', { button: 0 });
+        if (shift) {
+            cy.get('.cvat-canvas-container').trigger('mousemove', x, y, { shiftKey: true });
+        } else {
+            cy.get('.cvat-canvas-container').trigger('mousemove', x, y);
+        }
+        cy.get('#cvat_canvas_text_content').should('contain.text', expectedRorate);
+        cy.get('.cvat-canvas-container').trigger('mouseup');
+        cy.get(shape).should('have.attr', 'transform');
+    }
+
+    before(() => {
+        cy.openTask(taskName);
+        cy.openJob();
+        Cypress.config('scrollBehavior', false);
+        cy.createRectangle(createRectangleShape2Points);
+        cy.createRectangle(createRectangleTrack2Points);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Check that bounding boxes can be rotated.', () => {
+            testShapeRotate('#cvat_canvas_shape_1', 350, 150, '11.4°');
+            testShapeRotate('#cvat_canvas_shape_2', 350, 150, '26.6°');
+            cy.get('#cvat_canvas_shape_2').then((shape2) => {
+                const shapeAttrs = shape2.attr('transform').split('(')[1].split(')')[0].split(',');
+                for (const i of shapeAttrs) {
+                    transformMatrixShape.push(Number(i).toFixed(3));
+                }
```

Comment:
Try to use getCTM() instead.

Reply:
Try to use invoke. getCTM is a DOM method, I am not sure how to call it in cypress, but maybe with ``invoke``

Reply:
Unfortunately, I haven't found a solution yet for ```getCTM()```.
I have attempted to convert the matrix values from attr to ```Number``` values due to a slight change in these values if the test is run in the browser in headless mode. But now I've redone it to use values from css. This solution works well.